chpl/chpl-service/src/main/java/gov/healthit/chpl/FeatureList.java

```diff
@@ -1,15 +1,19 @@
 package gov.healthit.chpl;
 
 /**
- * All FF4J feature flags.
+ * All relevant FF4J feature flags.
  * @author alarned
  *
  */
-public class FeatureList {
+public final class FeatureList {
+    private FeatureList() {}
+
     /** System for Developers to submit change requests. */
     public static final String CHANGE_REQUEST = "change-request";
     /** Features that will be turned on as of the effective rule date. */
     public static final String EFFECTIVE_RULE_DATE = "effective-rule-date";
+    /** Features that will be turned on as of one week past the effective rule date. */
+    public static final String EFFECTIVE_RULE_DATE_PLUS_ONE_WEEK = "effective-rule-date-plus-one-week";
     /** Ability to have Developers as User. */
     public static final String ROLE_DEVELOPER = "role-developer";
-}
+}
```

Comment:
I think checkstyle doesn't like files that don't end with a newline, but I could be wrong

Reply:
Agreed - noticed that but left as is to match the way it was in the remote repo (no newline) so there is no merge conflict since there are changes from different people within the file.  I suppose that was pointless as git should be able to handle that.

Reply:
But you didn't leave it the way it was, because git is showing it as a change from having a newline to not having a newline. Otherwise it wouldn't have shown up as a changed line...

Reply:
Added and pushed